src/routers/productExports.ts

```diff
@@ -7,14 +7,110 @@ import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/**
+ * @api {get} /productexports
+ * @description Returns all product exports and all their associated attributes.
+ *              Guaranteed fields are:
+ *                - productTransactionUuid
+ *                - productType
+ *                - datetime (UTC)
+ *                - toPersonUuid
+ *                - fromPersonUuid
+ *                - amountOfProduct
+ *                - productUnits
+ *                - costPerUnit
+ *                - currency
+ *              Parameters have no effect on this request.
+ *              Associated attributes can be checked via /products/
```

Comment:
example of associate attribute for product exports? also i think its a different end point for product exports?